test/snoopi_deep.jl

```diff
@@ -944,6 +945,163 @@ end
     Pkg.activate(cproj)
 end
 
+_name(frame::SnoopCompileCore.InferenceTiming) = frame.mi_info.mi.def.name
+
+@testset "reentrant concurrent profiles 1 - overlap" begin
+    # Warmup
```

Comment:
Perhaps something like the following? :)
```suggestion
    # Warmup to prevent, e.g., `+` from appearing in the profiles; only `ROOT` and `foo*` should appear.
```

Reply:
(Likewise on the warmup comments below if you agree.)